.github/workflows/fixtures/modules/README.md

```diff
@@ -0,0 +1 @@
+This module is a copy coming from https://github.com/camunda/distribution/.
```

Comment:
Please clone from the team IaC repo instead of copying it here.


All that you need is to copy this step and clone using its token.
https://github.com/camunda/camunda-platform-helm/blob/95625f4295bcf141cf434a50f2005f7489aa00d9/.github/workflows/chart-update-readme.yaml#L22

Reply:
Add me as a reviewer once it's there :+1: